src/devices/CharacterLcd/LcdInterface.I2c.cs

```diff
@@ -59,7 +59,17 @@ public I2c(I2cDevice device)
 
             public override bool EightBitMode => true;
 
-            public override bool BacklightOn { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
+            public override bool BacklightOn 
+            {
+                get
+                {
+                    // Not implemented / Not supported, but don't throw exceptions here
```

Comment:
please explain why not in the comment (i.e. `we currently don't have a way of checking if backlight is supported and therefore exceptions will not be expected by the user code`)

Reply:
Updated the comment. It seems that with an 8 Bit I2C driver, this bit cannot be set anywhere, so the backlight is supposed to be on all the time (or is controlled externally).